pub-critic/src/app/components/GameCard/GameCard.tsx

```diff
@@ -0,0 +1,89 @@
+import { Game, Platform, PlatformFull } from "api/GamesShared";
+import classes from "./GameCard.module.scss";
+import Image from "next/image";
+import playstation from "assets/playstation.svg";
+import xbox from "assets/xbox.svg";
+import nintendo from "assets/nintendo.svg";
+import epic from "assets/epic.svg";
+import steam from "assets/steam.svg";
+import controller from "assets/controller.svg";
+interface GameCardProps {
+  game: Game;
+}
+
+const attachPlatformImage = (platforms: PlatformFull[]) => {
+  const images = [];
+  if (
+    platforms.find((p) => p.platform.name.toLowerCase().includes("playStation"))
+  ) {
+    images.push(playstation);
+  }
+  if (platforms.find((p) => p.platform.name === "PC")) {
+    images.push(steam);
+  }
+  if (platforms.find((p) => p.platform.name.toLowerCase().includes("xbox"))) {
+    images.push(xbox);
+  }
+  if (
+    platforms.find((p) => p.platform.name.toLowerCase().includes("nintendo"))
+  ) {
+    images.push(nintendo);
+  }
+  if (platforms.find((p) => p.platform.name.toLowerCase().includes("epic"))) {
+    images.push(epic);
+  }
+  return images;
+};
```

Comment:
1) tooooo many ifs, this must be simplified somehow in way that you find a platform name that matches certain platform -> `function` with "platform" `param`
2) move this to `app/utils/static`

Reply:
:white_check_mark: